build.gradle

```diff
@@ -27,20 +27,36 @@ android {
     sourceCompatibility JavaVersion.VERSION_1_7
     targetCompatibility JavaVersion.VERSION_1_7
   }
+
+  testOptions {
+    unitTests {
+      all {
+        jvmArgs '-noverify'
+      }
+    }
+  }
 }
 
 checkstyle {
   configFile rootProject.file('gradle/checkstyle.xml')
 }
 
+allprojects {
+  repositories {
+    maven {
+      url "https://comscore.bintray.com/Analytics"
```

Comment:
We should add this to the Readme. Customers will need to add this block to their Gradle configuration.